userguide/tutorials/user_management.adoc

```diff
@@ -125,37 +135,79 @@ But the following curl to refund a payment would fail because his role `customer
+org.killbill.security.ldap.permissionsByGroup=
+```
+
+Notes:
+
+* If no groups are defined in LDAP, all users will only have read-only permissions
+* Before an LDAP user can use Kaui, an admin needs to associate his login to the right tenants (see below)
 
-= KAUI
 
+== KAUI
 
 KAUI has been extended to understand all the user/role/permission management and will manage the corresponding sessions. Some of those implementation details were covered in http://killbill.io/blog/multi-tenancy-authorization[our previous blog post].
 
-As far as user configuration goes, this is all done on the server side and so KAUI does not export a way to add users, configure roles and associated permissions. As long as the user is correctly configured on the server side (either statically in `shiro.ini`, or in the database through the use of the apis we have shown previously, or by relying on an external LDAP configuration), the login process through KAUI should work.
+=== Users
 
-However, KAUI needs to be told about the relationship between user and tenants. This mapping is specific to the UI and on the server side, as long as the user connects using the right credentials, and specifying the right tenant `api_key` and `api_secret`, things will work. The idea, is that a super admin would first configure the allowed users for each specific tenant on the KAUI side, in such a way that later when a specific user logs in, he only sees the tenants he has access to.
+To configure users allowed to use Kaui, go to `/admin_allowed_users`:
 
-As far as tenant configuration goes, it is possible to create tenants directly from KAUI, or such tenants can also be created using apis (outside of KAUI). In the later case, KAUI needs to be told that such tenants exists (there is no auto-discovery); and then as discussed earlier mappings between tenants and allowed users need to be made inside KAUI.
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui/KAUI_NewAllowedUser.png[align=center]
 
-== Summary
+Fields to populate will depend on which back-end realm is configured.
 
-1. User, roles, permissions need to be configured on the server side (`shiro.ini`, database, LDAP)
-2. Tenants can then be created (using a user whole role's permissions allow such operation)
-3. Mapping between tenant and allowed users need to be defined in KAUI
+==== Database
+
+If you are storing roles, usernames and passwords in the Kill Bill database, leave the "Managed externally" checkbox unchecked and fill-in all details. Kaui will create these users locally and in Kill Bill (if they don't exist on the server already).
+
+Roles can be created by going to `/role_definitions/new`.
+
+==== LDAP and Okta
+
+If you are delegating roles and users management to a third-party system, Kaui only needs to know the login of the users. Password and roles being managed outside of Kill Bill, you need to access your third-party system to set and update these.
```

Comment:
Should we add that in this case the `Managed externally` box should be checked when creating users on KAUI ?